### ISSUE TITLE
Will this mod be combined with the Spore Biomechs mod 

### ISSUE THREAD

**Issue by Labbit35** [UnionofSovietSocialistRepublics/Sporebiomethane#3]
would it be better for the original mod to have these changes or is this a test mod and the completed version will be ported to the actual mod?
base sensitive question I build a factory in Birth

**Comment by UnionofSovietSocialistRepublics** [comment]
no, some changes on this version doesnt work on the hjson version, the hjson version willprobably only get balance changes or bug fixes from now on.

**Comment by Labbit35** [comment]
so the HJson version is the biomechs one right?
